pkgs/development/python-modules/python-manilaclient/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, buildPythonApplication, fetchPypi
+, pbr, oslo-config, oslo-log, oslo-serialization, oslo-utils, prettytable, requests, simplejson, Babel, osc-lib, python-keystoneclient, debtcollector
+, callPackage
+}:
+
+buildPythonApplication rec {
+  pname = "python-manilaclient";
+  version = "3.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "2d90af35c5beccc53fa6b0f5a3c4b330a065e86924c33c42b017f18943ab2b05";
+  };
+
+  nativeBuildInputs = [
+    pbr
+  ];
+
+  propagatedBuildInputs = [
+    oslo-config
+    oslo-log
+    oslo-serialization
+    oslo-utils
+    prettytable
+    requests
+    simplejson
+    Babel
+    osc-lib
+    python-keystoneclient
+    debtcollector
+  ];
+
+  doCheck = false;
```

Comment:
Please add the comment about infinite recursion. It is not that important which package causes it in openstack.